code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1422,6 +1422,10 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		if(BRUTE)
 			H.damageoverlaytemp = 20
 			var/damage_amount = forced ? damage : damage * hit_percent * brutemod * H.physiology.brute_mod
+			if(damage_amount > 5)
+				H.AdjustAllImmobility(damage_amount * -1/2 SECONDS)
```

Comment:
why did you regress this back to the fractions

Reply:
That was an accident oops, I'll fix it